### PR TITLE
[BUGFIX] slack darkmode not working due to forced bgcolor of elements

### PR DESCRIPTION
**Pull request by jpmschuler**
Seems Slack updated there HTML classes, thus `darkmode.css` needs to override new elements.
rels https://github.com/getferdi/ferdi/issues/1914

### CODE REVIEW

File: recipes/slack/darkmode.css [hunk 1]
```diff
@@ -1,6 +1,18 @@
 /* Copied from: https://github.com/ducfilan/Dark-mode-Franz-Ferdi */
 
-body {
```
Comment (vraravam) [inline recipes/slack/darkmode.css:3]: Whenever any file changes in a recipe, there's a validation check wherein the `version` in the `package.json` will also need to be bumped up. Please fix and test this with the command in the `CONTRIBUTING.md` file:
``` pmpn i && pnpm run package`

**Review by vraravam — CHANGES_REQUESTED** [review]

**Comment by jpmschuler** [comment]
done. 

FWIW: this bumping should probably be a github action instead in order to have version agnostic PRs.

Nevertheless additionally: the recommit hook throws errors:
```
> eslint . --fix
ferdi/recipes/recipes/toggl/webview-unsafe.js
  2:5   error  'ferdi' is not defined  no-undef
  2:27  error  'ferdi' is not defined  no-undef
  7:9   error  'ferdi' is not defined  no-undef
  9:9   error  'ferdi' is not defined  no-undef
```

**Review by vraravam — APPROVED** [review]
Thanks for the contribution @jpmschuler !

**Comment by kytwb** [comment]
@all-contributors please add @jpmschuler for code

**Pull request merged**